test/acceptance/view_haggadah_test.clj

```diff
@@ -75,20 +80,23 @@
         title actual-title
         bracha actual-bracha))))
 
-
-(def parsed-haggadah-text  "We begin in Egypt")
-
-(def new-haggadah-title "The best haggadah of the year")
+(def expected-haggadah-title "The best haggadah of the year")
 
 (t/deftest refresh-page-test
   (t/testing "When the current user refreshes the haggadah"
-
-      (c/create-haggadah {:title new-haggadah-title
-                          :content {:bracha {:content parsed-haggadah-text}}} "user1")
-      (doto driver
-        (c/home->dashboard)
-        (click-on-haggadah parsed-haggadah-text)
-        (e/refresh)
-        (e/wait-has-text-everywhere parsed-haggadah-text))
-      (let [haggadah-text (haggadah-content driver)]
-        (t/is (= parsed-haggadah-text haggadah-text)))))
+    (c/create-haggadah {:title expected-haggadah-title
```

Comment:
NIT: Maybe `store-haggadah` is a better name ... or `fs-store-haggadah` to make it clear it actually creates the document in Firestore

Reply:
I did not realize the name was unclear. I'll change the name.